src/webui/api/authcontroller.cpp

```diff
@@ -45,11 +45,15 @@ AuthController::AuthController(ISessionManager *sessionManager, IApplication *ap
 
 void AuthController::setUsername(const QString &username)
 {
+    if (!m_username.isEmpty() && m_username != username)
```

Comment:
Isn't `if (m_username != username)` enough?

Reply:
```suggestion
    if (!m_username.isEmpty() && (m_username != username))
```

Reply:
> Isn't `if (m_username != username)` enough?

I think it's not semantically correct to call `logoutAllSessions` during the initial setup of this controller, though at that time the session storage should be empty and calling the method should have no effect.